EngineLayer/FdrAnalysis/FdrAnalysisEngine.cs

```diff
@@ -87,7 +89,7 @@ private void DoFalseDiscoveryRateAnalysis(FdrAnalysisResults myAnalysisResults)
             //determine if Score or DeltaScore performs better
             if (UseDeltaScore)
             {
-                const double qValueCutoff = 0.01; //optimize to get the most PSMs at a 1% FDR
```

Comment:
Please leave the hard coded 0.01. The default qValue right now is 1.0, which doesn't make sense to use here, since delta and normal scores would produce the same results